### ISSUE TITLE
Matlab R2017a strings using double quotes

### ISSUE THREAD

**Issue by bebuch** [rouge-ruby/rouge#1047]
Starting with Matlab R2017a you can create strings using double quotes:

```matlab
x = "a string";    % some "ticks" in a comment
y = "a string with ""interal"" quotes";
```

This should be supported.

**Comment by pyrmont** [comment]
@bebuch So I take it from your example that two double quotes are interested as an escaped double quote? Are there other escape sequences relevant for double-quote strings?

**Comment by pyrmont** [comment]
This has been fixed by #1048.
